plugins/web/opentelemetry-instrumentation-long-task/src/instrumentation.ts

```diff
@@ -41,13 +31,17 @@ export class LongTaskInstrumentation extends InstrumentationBase {
   moduleName = this.component;
 
   private _observer?: PerformanceObserver;
+  private _observerCallback?: ObserverCallback;
```

Comment:
you don't have to store in as a private member.
the config object is accessible via `this._config.observerCallback` where you need to use it